website/docs/d/networkmanager_core_network_policy_document.html.markdown

```diff
@@ -12,7 +12,7 @@ Generates a Core Network policy document in JSON format for use with resources t
 
 Using this data source to generate policy documents is *optional*. It is also valid to use literal JSON strings in your configuration or to use the `file` interpolation function to read a raw JSON policy document from a file.
 
--> For more information about building AWS Core Network policy documents with Terraform, see the [Using AWS & AWSCC Provider Together Guide](https://learn.hashicorp.com/terraform/aws/<changeme>).
+-> For more information about building AWS Core Network policy documents with Terraform, see the [Using AWS & AWSCC Provider Together Guide](https://github.com/hashicorp/terraform-provider-aws/tree/main/website/docs/guides/using-aws-awscc-together).
```

Comment:
I don't think this is going to work as there is no such file.
Should it be https://learn.hashicorp.com/tutorials/terraform/using-aws-awscc-together?

Reply:
ack - i see what she meant now :D